src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1382,6 +1382,8 @@ internal enum ErrorCode
         ERR_TupleDuplicateMemberName = 8203,
         ERR_TupleExplicitNamesOnAllMembersOrNone = 8204,
 
-        ERR_PrototypeNotYetImplemented = 8204,
+        ERR_PredefinedTypeMemberNotFoundInAssembly = 8205,
+
+        ERR_PrototypeNotYetImplemented = 8206,
```

Comment:
You can simply assign number 9000 to this error, so that we don't have to adjust it every time we add new diagnostics